app/tests/test_mail.py

```diff
@@ -97,10 +100,14 @@ def test_notify_tenders_digest(self):
         soup = BeautifulSoup(alt_body, 'html.parser')
 
         tender_list = soup.find('ol', {'class': 'tender-list'}).find_all('a')
+        
+        tender_1_url = f"{settings.BASE_URL}/tenders/{self.tender1.id}"
+        tender_2_url = f"{settings.BASE_URL}/tenders/{self.tender2.id}"
+        teder_3_url = f"{settings.BASE_URL}/tenders/{self.tender3.id}"
         self.assertEqual(len(tender_list), 3)
-        self.assertEqual(tender_list[0]['href'], self.tender3.url)
-        self.assertEqual(tender_list[1]['href'], self.tender1.url)
-        self.assertEqual(tender_list[2]['href'], self.tender2.url)
+        self.assertEqual(tender_list[0]['href'], teder_3_url)
```

Comment:
Why is tender 3 the first in the list?

Reply:
The tenders are in a different order because in the HTML file, they are ordered by date of publication '-published'

In this test case, the URL for Tender 3 is stored in the tender_list[0]['href']

This is why the assertion statements for Tender 3, Tender 1, and Tender 2 are in that order, as they are checking that the URLs are correctly stored in the 'tender_list' list.